### PR TITLE
[PLAY-2] Add flexbox props to all kits

### PR DESCRIPTION
**Pull request by RobGentile17**
#### Screens
DESKTOP:
Screen Shot 2022-02-02 at 4 11 07 PM

MOBILE: 
Screen Shot 2022-02-02 at 4 11 46 PM

#### Breaking Changes

No.  There was only the addition of new props, nothing changed or removed that would affect existing code in nitro.

#### Runway Ticket URL

https://nitro.powerhrg.com/runway/backlog_items/PLAY-2

#### How to test this

alpha testing

#### Checklist:

- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [x] **SCREENSHOT** Please add a screen shot or two.
- [x] **SPECS** Please cover your changes with specs.
- [x] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

### CODE REVIEW

**Review by jasperfurniss — APPROVED** [review]

**Review by jasoncypret — CHANGES_REQUESTED** [review]

File: playbook/app/pb_kits/playbook/_playbook.scss [hunk 1]
```diff
@@ -100,3 +100,14 @@
...
+@import './utilities/justify_content';
+@import './utilities/flex_wrap';
+@import './utilities/justify_self';
+@import './utilities/align_items';
+@import './utilities/align_content';
+@import './utilities/align_self';
+@import './utilities/flex';
+@import './utilities/flex_grow';
+@import './utilities/flex_shrink';
+@import './utilities/order';
```
[the remark this answers is not in the document]
Comment (RobGentile17) [reply playbook/app/pb_kits/playbook/_playbook.scss]: I just updated it, is that what you meant? @jasoncypret

File: playbook/app/pb_kits/playbook/_playbook.scss [hunk 2]
```diff
@@ -100,3 +100,4 @@
 @import './utilities/shadow';
 @import './utilities/line_height';
 @import './utilities/display';
+@import './utilities/flexbox';
```
Comment (RobGentile17) [inline playbook/app/pb_kits/playbook/_playbook.scss:103]: @jasoncypret here is the new import

File: playbook/app/pb_kits/playbook/utilities/_flexbox.scss [hunk 3]
```diff
@@ -0,0 +1,11 @@
+@import './flex_direction';
```
Comment (RobGentile17) [inline playbook/app/pb_kits/playbook/utilities/_flexbox.scss:1]: @jasoncypret here's the new scss file

**Review by thestephenmarshall — APPROVED** [review]

**Comment by thestephenmarshall** [comment]
@RobGentile17 not sure about the conflict, but looks like it's time to GTM. 🚀

**Review by jasoncypret — APPROVED** [review]

**Review by kre8sions — COMMENTED** [review]
- We need to add !important to the scss flex styles.
- Also I noticed you are using flexStart and flexEnd if you are going to have those also add the normal start and end to the props where that is used @RobGentile17

**Pull request merged**